build_scripts/Dockerfile

```diff
@@ -141,6 +144,9 @@ FROM gcf-cpp-incremental-2 AS gcf-cpp-ci-0
 COPY . /usr/local/share/gcf
 RUN find /usr/local/share/gcf -type f | xargs chmod 644
 RUN find /usr/local/share/gcf -type d | xargs chmod 755
+# More recent versions of cmake error when attempting to run `configure_file` on
+# files owned by root. Changing the owner to user cnb fixes this.
```

Comment:
Q: what is cnb?

Reply:
'cnb' is the user and group created previously in the Dockerfile (line 40 ish) that is designed to be the active, non-root, user when the buildpack is executed.

Reply:
Thanks, missed that